admin-dev/themes/new-theme/js/components/form/disabling-toggler.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+/**
+ * Shows or hides specified element based on specified checkbox "checked" property.
+ */
+export default class DisablingToggler {
+  inputForMatchingSelector: string;
+
+  elementToToggleSelector: string;
+
+  matchingValue: string;
+
+  disableOnMatch: boolean;
+
+  constructor(
```

Comment:
Please document the parameters, or else we need to reverse engineer to understand how to use the method.
This has to be done at least for all public members (including return values) in all classes